encrypted-media-respec.html

```diff
@@ -318,7 +318,7 @@
           </p>
           <p>Each Key Session is associated with a single <a>MediaKeys</a> object, and only media element(s) associated with that object may access key(s) associated with the session.
           Other <a>MediaKeys</a> objects, <a def-id="cdm"></a> instances, and media elements MUST NOT access the key session or use its key(s).
-          Key sessions and the keys they contain are no longer <a def-id="usable-for-decryption"></a> when the <a def-id="session-close-algorithm">session is closed</a>, including when the <a>MediaKeySession</a> object is destroyed.
+          Key sessions and the keys they contain are no longer <a def-id="usable-for-decryption"></a> when the <a def-id="session-closed-algorithm">session is closed</a>, including when the <a>MediaKeySession</a> object is destroyed.
```

Comment:
The session-closed-algorithm reverence doesn't seem strictly correct since the keys are no longer usable before that algorithm is called. It might be more accurate to say "once the session has been closed". Note that the referenced algorithm is run "after a session has been closed."